contributor-growth/drafts/contributor-growth-framework/5-Incentivizing-non-code.md

```diff
@@ -0,0 +1,11 @@
+# 5. Incentivizing non-code contributions 
+
+While swag and awards can motivate community members to help others or share their stories, that strategy doesn't scale and is only feasible for small projects. It certainly is a good tool for projects earlier in their journey.  
+
+Consciously create an environment where learning in public is ok. This includes, public office hours, unboxing sessions, live streams where you figure out how to do something together.  
+
+Foster a Q&A culture where people feel comfortable to just jump in. Tag others familiar with a topic in conversations so they can chime in. This has two effects: 1) you publicly recognize the person as an expert and let them know that you value their opinion, and 2) everyone sees that it's not only the role of the maintainers to help out.  
+
+Especially for large projects, Slack can get messy quickly. If it's too noisy, people won't engage. Create topic specific Slack channels where it's easier to follow conversations they care about.  
+
+Non-code contributor roles such as moderators, community managers, and documentation maintainers should be part of your ladder. They can ask (loaded) questions in chats to seed discussions. Post helpful, curated content and links. While you can't ask people to answer questions, if the channel is not super high vol and/or specific to their interest, they'll answer.  :wq
```

Comment:
Looks like :wp snook in and this is missing as well:

To request content, consider creating a GitHub issue asking for blog posts. Contributors can send their links which you can tweet out. You'll need a community manager to keep track of this, though. 

## Long-term contributors

But you also want more permanent non-code contributions. Think tech writers, PXMs, or product managers, all have a great skill set your project can benefit from. Unfortunately, most of these people don't really know that they, too, can be active members of the open source community.

Make these roles known and give people a hat so they know their contribution is valuable. That will also help prove to their employer that their work is valued. Even if the hat feels ceremonial to you, it may give them the business case to work on that project. 
For these types of roles, the term "contributor ladder" isn't helpful. While for code contributions it really is more of a ladder where you get increasingly more responsibilities and privileges, these non-code contributor roles are more like different buckets with every different – but just as important – skill sets. 

For larger projects, different governance groups also fall under this category. The steering committee and various subcommittees all have very different skill sets that your project can benefit from.

Reply:
This has been added as well